04.ls/ls.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+COLUMN = 3
+SPACE = 1
+
+def main
+  current_path = '../../dummy/' # "#{Dir.getwd}/"
```

Comment:
好みの問題かもしれませんが、`current`以外のpathはないので、

```suggestion
  path = '../../dummy/' # "#{Dir.getwd}/"
```

とするのもありです。